### PR TITLE
[OSDOCS-5511]: Misc loose ends for CPMS in 4.13

### PR DESCRIPTION
**Pull request by jeana-redhat**
Version(s):
4.12+ but selectively (see _Additional information_)

Issue:
[OSDOCS-5511](https://issues.redhat.com//browse/OSDOCS-5511)

Link to docs preview:
* Ensured correct use of  "Control Plane Machine Set Operator" vs. "control plane machine set(s)" [throughout text and ToC entries
](https://57077--docspreview.netlify.app/openshift-enterprise/latest/machine_management/control_plane_machine_management/cpmso-about.html)
* Updated platform-specific refs in [Limitations](https://57077--docspreview.netlify.app/openshift-enterprise/latest/machine_management/control_plane_machine_management/cpmso-about.html#cpmso-limitations_cpmso-about) (platforms supported, GCP preemptible VM unsupported)
* In [Provider-specific configuration](https://57077--docspreview.netlify.app/openshift-enterprise/latest/machine_management/control_plane_machine_management/cpmso-configuration.html#cpmso-sample-yaml-provider-specific_cpmso-configuration), flipped "`<platform_failure_domains>` and `<platform_provider_spec>`" (spec first)
* Added GCP to [Failure domain platform support and configuration](https://57077--docspreview.netlify.app/openshift-enterprise/latest/machine_management/control_plane_machine_management/cpmso-resiliency.html#cpmso-failure-domains-provider_cpmso-resiliency), including additional resources link
* Made the long commands in [Troubleshooting the control plane machine set](https://57077--docspreview.netlify.app/openshift-enterprise/latest/machine_management/control_plane_machine_management/cpmso-troubleshooting.html) and [Disabling the control plane machine set](https://57077--docspreview.netlify.app/openshift-enterprise/latest/machine_management/control_plane_machine_management/cpmso-disabling.html) multi-line
* Updated [Control Plane Machine Set Operator](https://57077--docspreview.netlify.app/openshift-enterprise/latest/operators/operator-reference.html#control-plane-machine-set-operator_cluster-operators-ref) reference

QE review:
- [ ] QE has approved this change.

Additional information:
* ~~_Hold review for #56751 merge and rebase_~~
* Will require a manual CP into 4.12 to only grab the relevant parts

### CODE REVIEW

**Review by JoelSpeed — COMMENTED** [review]
LGTM

**Comment by jeana-redhat** [comment]
@sunzhaohua2 This PR cleans up some terminology, and addresses some missed updates for 4.13. I will backport it selectively into 4.12 (to get the terminology updates, but not the GCP content). Can you PTAL for QA?

File: modules/cpmso-creating-cr.adoc [hunk 1]
```diff
@@ -58,15 +58,15 @@ spec:
...
 +
 [IMPORTANT]
 ====
 Before you activate the CR, you must ensure that its configuration is correct for your cluster requirements.
 ====
-<3> Specify the update strategy for the cluster. The allowed values are `OnDelete` and `RollingUpdate`. The default value is `RollingUpdate`. 
+<3> Specify the update strategy for the cluster. The allowed values are `OnDelete` and `RollingUpdate`. The default value is `RollingUpdate`.
 //Todo: For more information about update strategies, see "Updating the control plane configuration".
-<4> Specify your cloud provider platform name. The allowed values are `AWS`, `Azure`, and `VSphere`. 
+<4> Specify your cloud provider platform name. The allowed values are `AWS`, `Azure`, and `VSphere`.
```
Comment (sunzhaohua2) [inline modules/cpmso-creating-cr.adoc:69]: missed `GCP`

**Comment by sunzhaohua2** [comment]
Thanks @jeana-redhat, one comment, others all lgtm.

**Pull request merged**